extension/js/common/core/crypto/pgp/msg-util.ts

```diff
@@ -145,7 +143,14 @@ export class MsgUtil {
         // This does not 100% mean it's OpenPGP message
         // But it's a good indication that it may be
         const t = opgp.enums.packet;
-        const msgTpes = [t.symEncryptedIntegrityProtected, t.modificationDetectionCode, t.symEncryptedAEADProtected, t.symmetricallyEncrypted, t.compressed];
+        const msgTpes = [
+          t.publicKeyEncryptedSessionKey,
```

Comment:
I added `publicKeyEncryptedSessionKey` type here as a pubkey-encrypted message started with this type of packet. So this method isn't used often and tested properly.
We can review and test it in a separate issue

Reply:
ok

Reply:
I don't think we necessarily have to unit-test this method, it's not like we normally edit it or plan to do so. I think it's likely broadly covered by other tests.